molgenis-data/src/test/java/org/molgenis/data/support/HandlebarsMathHelperTest.java

```diff
@@ -0,0 +1,136 @@
+package org.molgenis.data.support;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import com.github.jknack.handlebars.Handlebars;
+import com.github.jknack.handlebars.HandlebarsException;
+import com.github.jknack.handlebars.Template;
+import java.io.IOException;
+import org.junit.jupiter.api.Test;
+import org.molgenis.test.AbstractMockitoTest;
+
+class HandlebarsMathHelperTest extends AbstractMockitoTest {
```

Comment:
I think I'm missing some test cases for where the attributes are string or some other type that can't be "math'd upon" :)